prototype_source/pt2e_quant_ptq_static_x86_inductor.rst

```diff
@@ -0,0 +1,187 @@
+    —--------------------------------------------------------
+                                |
+                             Inductor
+
+Combining Quantization in PyTorch 2.0 Export and TorchInductor, we have flexibility and productivity with the new Quantization frontend
+and outstanding out-of-box performance with the compiler backend. Especially on Intel fourth generation (SPR) Xeon processors which can
+further boost the models' performance by leveraging the
+`advanced-matrix-extensions <https://www.intel.com/content/www/us/en/products/docs/accelerator-engines/advanced-matrix-extensions/overview.html>`_ feature.
+
+Now, we will walk you through a step-by-step tutorial for how to use it with `torchvision resnet18 model <https://download.pytorch.org/models/resnet18-f37072fd.pth>`_.
+
+1. Capture FX Graph
+---------------------
+
+We will start by performing the necessary imports, capturing the FX Graph from the eager module.
+
+::
+
+    import torch
+    import torchvision.models as models
+    import copy
+    from torch.ao.quantization.quantize_pt2e import prepare_pt2e, convert_pt2e
+    import torch.ao.quantization.quantizer.x86_inductor_quantizer as xiq
+    from torch.ao.quantization.quantizer.x86_inductor_quantizer import X86InductorQuantizer
+    from torch._export import capture_pre_autograd_graph
+
+    # Create the Eager Model
+    model_name = "resnet18"
+    model = models.__dict__[model_name](pretrained=True)
+
+    # Set the model to eval mode
+    model = model.eval()
+
+    # Create the data, using the dummy data here as an example
+    traced_bs = 50
+    x = torch.randn(traced_bs, 3, 224, 224).contiguous(memory_format=torch.channels_last)
+    example_inputs = (x,)
+
+    # Capture the FX Graph to be quantized
+    with torch.no_grad():
+         # if you are using the PyTorch nightlies or building from source with the pytorch master,
+        # use the API of `capture_pre_autograd_graph`
+        # Note 1: `capture_pre_autograd_graph` is also a short-term API, it will be updated to use the official `torch.export` API when that is ready.
+        exported_model = capture_pre_autograd_graph(
+            model,
+            example_inputs
+        )
+        # Note 2: if you are using the PyTorch 2.1 release binary or building from source with the PyTorch 2.1 release branch,
+        # please use the API of `torch._dynamo.export` to capture the FX Graph.
+        # exported_model, guards = torch._dynamo.export(
+        #     model,
+        #     *copy.deepcopy(example_inputs),
+        #     aten_graph=True,
+        # )
+
+
+Next, we will have the FX Module to be quantized.
+
+2. Apply Quantization
+----------------------------
+
+After we capture the FX Module to be quantized, we will import the Backend Quantizer for X86 CPU and configure how to
+quantize the model.
+
+::
+
+    quantizer = X86InductorQuantizer()
+    quantizer.set_global(xiq.get_default_x86_inductor_quantization_config())
```

Comment:
I think one additional thing is a doc for `X86InductorQuantizer`, that explains how user can use this, doesn't have to be added here in this doc though. I think it could live in pytorch or in intel codebase/docs, we can discuss which makes more sense

Reply:
Thanks, Make sense. I have added it as TODO and we can further discuss for where to put the doc of `X86InductorQuantizer`.

Reply:
OK thanks